Assets/_Laz/Tests/PlanetoidTests.cs

```diff
@@ -58,6 +58,96 @@ public void Test_Planetoid_Reset()
             Assert.AreEqual(mockPlanetoidBehaviour.ResetCount, 1, "Reset should be called once");
         }
 
+        [Test]
+        public void Test_Planetoid_WrappableManager_TriggersWin()
+        {
+            //Arrange
+            var numMockPlanets = 5;
+            MockPlanetoidBehaviour[] mockPlanetoids = new MockPlanetoidBehaviour[numMockPlanets];
+            Planetoid[] planetoids = new Planetoid[numMockPlanets];
+            for(int i = 0; i < numMockPlanets; i++)
+            {
+                mockPlanetoids[i] = new MockPlanetoidBehaviour();
+                planetoids[i] = new Planetoid(mockPlanetoids[i]);
+                mockPlanetoids[i].PlanetoidModel = planetoids[i];
+            }
+            var mockStateManager = new MockStateManager();
+            var lazoWrappableManager = new LazoWrappableManager(mockPlanetoids, mockStateManager);
+
+            //Act
+            mockStateManager.SetState(State.Play);
+            for (int i = 0; i < numMockPlanets; i++)
+            {
+                planetoids[i].ActivateLazo();
+            }
+
+            //Assert
+            Assert.AreEqual(mockStateManager.GetState(), State.WinGame, "State should be set to win");
+            for (int i = 0; i < numMockPlanets; i++)
+            {
+                Assert.IsTrue(planetoids[i].IsActivated, $"Planetoid {i} should be activated");
+            }
+        }
+
+        [Test]
+        public void Test_Planetoid_WrappableManager_CleanUp()
+        {
+            // Arrange
+            var numMockPlanets = 5;
+            MockPlanetoidBehaviour[] mockPlanetoids = new MockPlanetoidBehaviour[numMockPlanets];
+            Planetoid[] planetoids = new Planetoid[numMockPlanets];
+            for (int i = 0; i < numMockPlanets; i++)
+            {
+                mockPlanetoids[i] = new MockPlanetoidBehaviour();
+                planetoids[i] = new Planetoid(mockPlanetoids[i]);
+                mockPlanetoids[i].PlanetoidModel = planetoids[i];
+            }
+            var mockStateManager = new MockStateManager();
+            var lazoWrappableManager = new LazoWrappableManager(mockPlanetoids, mockStateManager);
+
+            //Act
+            mockStateManager.SetState(State.Play);
+            for (int i = 0; i < numMockPlanets; i++)
+            {
+                planetoids[i].ActivateLazo();
+            }
+            lazoWrappableManager.CleanUp();
+
+            //Assert
+            Assert.AreEqual(mockStateManager.GetState(), State.WinGame, "State should be set to win");
```

Comment:
You don't need to assert this, since the `CleanUp` Function doesn't really affect this.

Reply:
In this case I was trying to copy the format of the Planetoid_CleanUp test, as that first tests that the subscriber is called and then checks that the planetoid isn't activated, but yeah I'll remove this